### PR TITLE
feat: history page with imgs

### PR DESCRIPTION
**Pull request by tim08094495757**
## Types of changes
* **New feature**

## Description
blocked by https://github.com/pycontw/pycontw-frontend/pull/240

## Steps to Test This Pull Request
1. open /2022/about/history
2. check history's text: [zh-hant, tw]
3. check images' by adjusting the screen width 

## Related Issue
#232

### CODE REVIEW

File: pages/about/history.vue [hunk 1]
```diff
@@ -14,165 +14,183 @@
                 </p>
             </template>
         </banner>
```
Comment (tim08094495757) [inline pages/about/history.vue:16]: banner: wait for https://github.com/pycontw/pycontw-frontend/pull/240

**Comment by mattwang44** [comment]
please help resolve the conflict.

**Review by flynnhou — APPROVED** [review]
lgtm

**Comment by flynnhou** [comment]
#266 (refactored from #240) will update the banner without affecting this PR.

File: pages/about/history.vue (hunk 1 above)
Comment (flynnhou) [reply pages/about/history.vue:16]: #266 (refactored from #240) will update the banner without affecting this PR.

File: pages/about/history.vue [hunk 2]
```diff
@@ -235,7 +271,20 @@ export default {
...
+    font-size: 28px;
+}
+.history__paragraph-content {
+    @apply mt-1 mb-9 leading-5 md:leading-8 last:mb-0;
+    font-size: 16px;
+}
+@media (min-width: 768px) {
+    .history__paragraph-title {
+        font-size: 32px;
+    }
```
Comment (mattwang44) [inline pages/about/history.vue:285]: spec updated during design reviewing

```suggestion
.history__paragraph-title {
    @apply mt-1 mb-1 font-serif font-bold text-left text-pink-700;
    font-size: 24px;
}
.history__paragraph-content {
    @apply mt-1 mb-9 leading-5 md:leading-8 last:mb-0;
    font-size: 16px;
}
@media (min-width: 768px) {
    .history__paragraph-title {
        font-size: 28px;
    }
```

**Pull request merged**